core/src/main/scala/kafka/server/Server.scala

```diff
@@ -29,6 +29,7 @@ trait Server {
 }
 
 object Server {
+  val metadataTopicName = "@metadata"
```

Comment:
Is this used anywhere?

Reply:
RaftClient requires us to pass this as an argument, for now at least

Reply:
Can we use `KafkaRaftServer.MetadataTopic` and remove this?

Reply:
yes, let's do that